.travis.yml

```diff
@@ -13,26 +12,34 @@
 # limitations under the License.
 
 sudo: required
-
-language: python
-
-python:
-  - "3.6"
-
-services:
-  - docker
-
-before_install:
-  - ./install/scripts/docker_setup_forseti.sh
-
-install:
-  -
-
-script:
-  - ./install/scripts/docker_run_forseti.sh
-  - ./install/scripts/docker_unittest_forseti.sh
-  - ./install/scripts/docker_pystyle_forseti.sh
-
-after_success:
-  # Collect codecov results.
-  - docker -l error exec -it build /bin/bash -c "bash <(curl -s https://codecov.io/bash)"
+jobs:
+  include:
+  - stage: Integration test
+    name: Integration test - docker
+    services:
+    - docker
+    before_install:
+    - openssl aes-256-cbc -K $encrypted_5c0ae2aa862c_key -iv $encrypted_5c0ae2aa862c_iv
+      -in secret-key.tar.enc -out secret-key.tar -d
+    - tar xvf secret-key.tar
+    install:
+    - docker image pull gcr.io/cloud-foundation-cicd/cft/developer-tools:0.2.0
```

Comment:
Why static reference to a specific version?  What happens the images is updated?  Should we just use the latest?

Reply:
I think we should point to the docker image that works for us and not use the latest one as there might be breaking changes. Also, I don't think there is a `latest` tag.